cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/autocomplete_switch.spec.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  INDEX_WITH_TIME_1,
+  INDEX_WITH_TIME_2,
+  SECONDARY_ENGINE,
+  QueryLanguages,
+} from '../../../../../utils/constants';
+import {
+  getRandomizedWorkspaceName,
+  getRandomizedDatasourceName,
+  getDefaultQuery,
+} from '../../../../../utils/apps/query_enhancements/shared';
+import {
+  generateAutocompleteTestConfigurations,
+  LanguageConfigs,
+  getDatasetName,
+} from '../../../../../utils/apps/query_enhancements/autocomplete';
+
+const workspaceName = getRandomizedWorkspaceName();
+const datasourceName = getRandomizedDatasourceName();
+
+const generateAutocompleteTestConfiguration = (dataset, datasetType, language) => {
+  const baseConfig = {
+    dataset,
+    datasetType,
+    language: language.name,
+    testName: `${language.name}-${datasetType}`,
+  };
+
+  return {
+    ...baseConfig,
+  };
+};
+
+export const runAutocompleteTests = () => {
+  describe('discover autocomplete tests', () => {
+    beforeEach(() => {
+      cy.setupTestData(
+        SECONDARY_ENGINE.url,
```

Comment:
just fyi I am changing this to `PATHS.SECONDARY_ENGINE_URL` or something like that in my other PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/9319

but if your PR gets merged first, i will take care of it (Sorry for redundant comments, just moreso leaving a note to myself)

Reply:
👍